src/axom/core/utilities/FileUtilities.cpp

```diff
@@ -120,6 +122,9 @@ void getDirName(std::string& dir, const std::string& path)
   }
 }
 
+//-----------------------------------------------------------------------------
+void removeFile(const std::string& filename) { Unlink(filename.c_str()); }
```

Comment:
Should this return an error code? e.g. 0 for success, nonzero for failure?
See:
* https://man7.org/linux/man-pages/man2/unlink.2.html
* https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/unlink-wunlink?view=msvc-170

Reply:
It returns a return value now, passing along unlink's return value.